modules/seqexec/server/src/main/scala/seqexec/server/tcs/TcsConfigRetriever.scala

```diff
@@ -20,7 +20,14 @@ import seqexec.model.TelescopeGuideConfig
 import seqexec.server.EpicsCodex.{DecodeEpicsValue, decode}
 import seqexec.server.tcs.TcsController.FollowOption.{FollowOff, FollowOn}
 import seqexec.server.SeqexecFailure
-import seqexec.server.gems.Gems.GemsWfsStatus
+import seqexec.server.gems.Gems.{DetectorStateOps, GemsWfsState, Ngs1DetectorState, Ngs2DetectorState, Ngs3DetectorState, Odgw1DetectorState, Odgw2DetectorState, Odgw3DetectorState, Odgw4DetectorState}
+import seqexec.server.gems.Gems.Ngs1DetectorState.{Off => _, On => _, _}
+import seqexec.server.gems.Gems.Ngs2DetectorState.{Off => _, On => _, _}
+import seqexec.server.gems.Gems.Ngs3DetectorState.{Off => _, On => _, _}
+import seqexec.server.gems.Gems.Odgw1DetectorState.{Off => _, On => _, _}
+import seqexec.server.gems.Gems.Odgw2DetectorState.{Off => _, On => _, _}
+import seqexec.server.gems.Gems.Odgw3DetectorState.{Off => _, On => _, _}
+import seqexec.server.gems.Gems.Odgw4DetectorState.{Off => _, On => _, _}
```

Comment:
Are these imports to pick up the implicits? I thought that would happen by default.